rust-src/concordium_base/src/id/account_holder.rs

```diff
@@ -954,6 +954,7 @@ pub fn create_unsigned_credential<
     Ok((info, commitment_rands))
 }
 
+/// compute-proof-of-knowledge-signature
```

Comment:
Compute proof of knowledge signature
```suggestion
/// Compute proof of knowledge signature
```